.travis.yml

```diff
@@ -10,11 +10,12 @@ addons:
       # Needed for NetMHC
       - tcsh
 cache:
-  pip: true
-  # cache directory used for Ensembl downloads of GTF and FASTA files
+  # cache pip files, also directory used for Ensembl downloads of GTF and FASTA files
```

Comment:
looks like that hasn't been updated since March of this year, and we may as well use the most recent Travis docs?